tools/gluten-it/src/main/scala/io/glutenproject/integration/tpc/ds/TpcdsSuite.scala

```diff
@@ -24,9 +25,9 @@ class TpcdsSuite(
   val shufflePartitions: Int,
   val minimumScanPartitions: Boolean) extends TpcSuite(
   actions, testConf, baselineConf, fixedWidthAsDouble,
-  logLevel, errorOnMemLeak, enableUi, enableHsUi, hsUiPort, cpus,
-  offHeapSize, disableAqe, disableBhj, disableWscg, shufflePartitions,
-  minimumScanPartitions) {
+  logLevel, errorOnMemLeak, enableUi, enableHsUi, enableCeleborn,
+  hsUiPort, cpus, offHeapSize, disableAqe, disableBhj, disableWscg,
+  shufflePartitions, minimumScanPartitions) {
```

Comment:
Let's make `enableHsUi` and `hsUiPort` conjective.

Reply:
@zhztheplayer done.